dotnet/samples/Concepts/Prompty/PromptyFunction.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel;
+
+namespace Prompty;
+
+public class PromptyFunction(ITestOutputHelper output) : BaseTest(output)
+{
+    [Fact]
+    public async Task InlineFunctionAsync()
+    {
+        Kernel kernel = Kernel.CreateBuilder()
+            .AddOpenAIChatCompletion(
+                modelId: TestConfiguration.OpenAI.ChatModelId,
+                apiKey: TestConfiguration.OpenAI.ApiKey)
+            .Build();
+
+        string promptTemplate = @"---
+name: Contoso_Chat_Prompt
+description: A sample prompt that responds with what Seattle is.
+authors:
+  - ????
+model:
+  api: chat
+  configuration:
+    type: openai
+---
+system:
+You are a helpful assistant who knows all about cities in the USA
+
+user:
+What is Seattle?
+";
+
+        var function = kernel.CreateFunctionFromPrompty(promptTemplate);
+
+        var result = await kernel.InvokeAsync(function);
+        Console.WriteLine(result);
+    }
+}
```

Comment:
If memory serves, there were a bunch of prompty files checked in as part of tests. Can those now just be in-line in the tests?

Reply:
They're long but I can change the tests to load them as embedded resources instead.